content/en/serverless/step_functions/installation.md

```diff
@@ -224,11 +157,9 @@ If you have not yet instrumented your Lambda functions to send traces, you can [
 {{% /tab %}}
 {{< /tabs >}}
 
-## Enable enhanced metrics
-
-Datadog generates [enhanced metrics][8] from collected Cloudwatch logs. To enable this, add a `DD_ENHANCED_METRICS` tag to each of your Step Functions and set the value to `true`.
+## Link Step Functions with your AWS Lambda traces
```

Comment:
Minor: I think this section should ideally be part of the `Enable tracing` section. Or maybe put it after the `Enable Tracing` section.